src/app/public/modules/modal-provider/types/modal-provider.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  EventEmitter
+} from '@angular/core';
+
+import {
+  Observable
+} from 'rxjs';
+
+import {
+  SkyModalProviderCloseArgs
+} from './modal-provider-close-args';
+
+export interface SkyModalProvider {
+
+  type: string;
```

Comment:
Is there room to define/register the specific types so the consumers know what's available? Maybe something we can discuss during Flex.

Reply:
We can discuss it but I would personally lean towards not registering that as we would have to constantly manage that and I've not had luck in the past trying to manage something like that.